profile.nix

```diff
@@ -0,0 +1,41 @@
+{ config, pkgs, ... }:
```

Comment:
This looks like an example file that should be removed or moved to the documentation.